README.md

```diff
@@ -41,6 +41,23 @@ described earlier.
     cargo test
 
 
+Travis
+------
+CSearch is continuously integrated on [Travis CI].
+
+To update encrypted credentials in the `.travis.yml` file, use the
+[Travis command line tool].
+
+    gem install travis
```

Comment:
I've merged the travis packages from that container definition into the Dockerfile in this repo now.